crates/trigger/src/runtime_config.rs

```diff
@@ -198,8 +198,8 @@ pub struct RuntimeConfigOpts {
     #[serde(default)]
     pub llm_compute: Option<LlmComputeOpts>,
 
-    #[serde(rename = "config_provider", default)]
-    pub config_providers: Vec<ConfigProviderOpts>,
+    #[serde(alias = "config_provider", rename = "variables_provider", default)]
```

Comment:
super nit: consider swapping the order so the "main" name is first 

```suggestion
    #[serde(rename = "variables_provider", alias = "config_provider", default)]
```